src/functions/Coverage.ps1

```diff
@@ -75,9 +75,9 @@ function New-CoverageInfo {
 function Get-CoverageInfoFromDictionary {
     param ([System.Collections.IDictionary] $Dictionary)
 
-    [string] $path = Get-DictionaryValueFromFirstKeyFound -Dictionary $Dictionary -Key 'Path', 'p'
-    if ([string]::IsNullOrEmpty($path)) {
-        throw "Coverage value '$Dictionary' is missing required Path key."
+    $path = Get-DictionaryValueFromFirstKeyFound -Dictionary $Dictionary -Key 'Path', 'p'
+    if ($null -eq $path -or 0 -ge @($path).Count) {
```

Comment:
Would this be more readable whilst still being the same?
```suggestion
    if ($null -eq $path -or @($path).Count -eq 0) {
```

Reply:
It is not totally the same, and I do (or should do) these `none` and `any` checks always this way. But yes it would be more readable, but not more reliable.

Reply:
How is it different? As far as I know PowerShell always populates .Count and is therefore always >=0